server/src/config.ts

```diff
@@ -1,10 +1,15 @@
 import { ServiceConfigurationOptions } from 'aws-sdk/lib/service';
 
+// Listo Server options
+export const port = process.env.LISTO_SERVER_PORT || 8000;
+
+// Dynamo Config options
 export const region = 'ap-southeast-2';
 export const tableName = process.env.DYNAMODB_TABLE || 'Boards';
-export const debug = process.env.DEBUG || false;
-
 export const dynamoConfigOptions: ServiceConfigurationOptions = {
   region: region,
   endpoint: process.env.DYNAMO_DB_ENDPOINT,
 };
+
+// Disk database options
+export const diskPath = process.env.DISK_PATH || './db.json';
```

Comment:
I was thinking we should try to keep all config in a handful of places so that it makes configuring Listo easier for someone not familiar with the codebase? Otherwise we would also put the Dynamo config in the Dynamo file and remove the config file completely? Open to doing this though.

Reply:
I tend to agree with @zsims, it feels odd to specify config for this and then hardcode it inside the `Disk` class. 

I would recommend passing a config object to the constructor, eg `new Disk({ path })`. If not path is passed, the default path should live inside `Disk`. I would do something similar for the DynamoDB constructor (i.e. pass a config object with table name/region/etc)

Reply:
Ok gave it a go, let me know if thats what you were thinking.